catalog/core/catalog-core-camelcomponent/src/main/java/ddf/camel/component/catalog/content/ContentProducerDataAccessObject.java

```diff
@@ -192,25 +189,15 @@ public void createContentItem(FileSystemPersistenceProvider fileIdMap, ContentEn
 
     public void processHeaders(Map<String, Object> headers, StorageRequest storageRequest,
             File ingestedFile) {
-        String attributeOverrideHeaders = (String) headers.get(Constants.ATTRIBUTE_OVERRIDES_KEY);
-        if (StringUtils.isNotEmpty(attributeOverrideHeaders)) {
+        HashMap<String, Serializable> attributeOverrideHeaders = new HashMap<>((Map) headers.get(
```

Comment:
I am casting to the `Map` interface. Unless you mean to declare it as a `Map` (which isn't Serializable so I wouldn't be able to pass it into the storageRequest properties)